src/battle/action_cmd.h

```diff
@@ -0,0 +1,175 @@
+#ifndef _BATTLE_ACTION_CMD_H_
+#define _BATTLE_ACTION_CMD_H_
+
+#include "common.h"
+#include "hud_element.h"
+#include "battle/action_cmd/jump.h"
+#include "battle/action_cmd/hammer.h"
+#include "battle/action_cmd/flee.h"
+#include "battle/action_cmd/break_free.h"
+#include "battle/action_cmd/whirlwind.h"
+#include "battle/action_cmd/stop_leech.h"
+#include "battle/action_cmd/07.h"
+#include "battle/action_cmd/dizzy_shell.h"
+#include "battle/action_cmd/fire_shell.h"
+#include "battle/action_cmd/0A.h"
+#include "battle/action_cmd/bomb.h"
+#include "battle/action_cmd/body_slam.h"
+#include "battle/action_cmd/air_lift.h"
+#include "battle/action_cmd/air_raid.h"
+#include "battle/action_cmd/squirt.h"
+#include "battle/action_cmd/power_shock.h"
+#include "battle/action_cmd/mega_shock.h"
+#include "battle/action_cmd/smack.h"
+#include "battle/action_cmd/spiny_surge.h"
+#include "battle/action_cmd/hurricane.h"
+#include "battle/action_cmd/spook.h"
+#include "battle/action_cmd/water_block.h"
+#include "battle/action_cmd/tidal_wave.h"
+
+enum MashMeterColorModes {
+    MASH_METER_MODE_BLINK       = -1,
+    MASH_METER_MODE_MULTI_COLOR = 0,
+    MASH_METER_MODE_ONE_COLOR   = 1,
+};
+
+enum ActionTutorialModes {
+    ACTION_TUTORIAL_LEARN_BLOCK         = -1,
+    ACTION_TUTORIAL_MOVES_NOT_LEARNED   = 0,
+    ACTION_TUTORIAL_MOVES_LEARNED       = 1,
+    ACTION_TUTORIAL_GUIDED              = 2,
+    ACTION_TUTORIAL_WAIT_FOR_INPUT      = 3,
+};
+
+typedef struct ActionCommandStatus {
+    /* 0x00 */ s32 dynEntityId;
```

Comment:
You can call this workerID. Theres some confusion in the codebase about generic_entity / dynamic_entity... in the end, I think we're going to rename those to workers.